packages/url/src/filter-url-for-display.js

```diff
@@ -19,5 +19,26 @@ export function filterURLForDisplay( url ) {
 		return filteredURL.replace( '/', '' );
 	}
 
+	// Prettify image urls
+	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/g;
+	const colon = ':';
+	const period = '.';
+	const query = '?';
+	if ( filteredURL.match( mediaRegexp ) ) {
+		let tokens = filteredURL.split( '/' );
+		let imageToken = tokens[ tokens.length - 1 ];
+		if ( imageToken.includes( query ) ) {
+			imageToken = imageToken.split( query )[ 0 ];
+		}
+		if ( tokens[ 0 ].includes( colon ) ) {
+			tokens = tokens[ 0 ].split( colon );
+		}
+		if ( tokens[ 0 ].includes( period ) ) {
+			tokens = tokens[ 0 ].split( period );
+		}
+		const prettifiedImageURL = tokens[ 0 ] + '...' + imageToken;
```

Comment:
Seems an ellipsis character `…` would be most appropriate here.

Related:
- https://en.wikipedia.org/wiki/Ellipsis
- https://github.com/WordPress/gutenberg/blob/7e964a87aaf930c42f56272eb39f6076ea6f8fc4/.eslintrc.js#L88-L92